File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/LaundryRoomSimple.java
package com.pennapps.labs.pennmobile.classes;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by Jackie on 2017-10-21.
 */

public class LaundryRoomSimple {

    @SerializedName("hall_name")
    @Expose
    public String name;
    @SerializedName("id")
    @Expose
    public Integer id;
    @SerializedName("location")
    @Expose
    public String location;

}

File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/LaundryRoomSimple.kt
package com.pennapps.labs.pennmobile.classes

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by Varun on 10/25/2018.
 */

class LaundryRoomSimple {

    @SerializedName("hall_name")
    @Expose
    @JvmField var name: String? = null
    @SerializedName("id")
    @Expose
    @JvmField var id: Int? = null
    @SerializedName("location")
    @Expose
    @JvmField var location: String? = null

}
